api/envoy/extensions/transport_sockets/quic/v3/quic_transport.proto

```diff
@@ -20,6 +22,10 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message QuicDownstreamTransport {
   tls.v3.DownstreamTlsContext downstream_tls_context = 1
       [(validate.rules).message = {required: true}];
+
+  // If false, QUIC will tell TLS to reject any early data and to stop issuing 0-RTT credentials with resumption session tickets. So that the clients won't attempt 0-RTT requests.
```

Comment:
```suggestion
  // If false, QUIC will tell TLS to reject any early data and to stop issuing 0-RTT credentials with resumption session tickets. This will prevent clients from attempting 0-RTT requests.
```